src/NetAnalyzers/Microsoft.CodeAnalysis.NetAnalyzers.md

```diff
@@ -2070,6 +2070,18 @@ A call to 'Stream.Read' may return fewer bytes than requested, resulting in unre
 |CodeFix|True|
 ---
 
+## [CA2023](https://learn.microsoft.com/dotnet/fundamentals/code-analysis/quality-rules/ca2023): Invalid braces in message template
+
+The braces present in the message template are invalid.  Ensure any braces in the message template are valid opening/closing braces, or are escaped.
+
+|Item|Value|
+|-|-|
+|Category|Reliability|
+|Enabled|True|
+|Severity|Error|
```

Comment:
>Error [](http://example.com/codeflow?start=10&length=5)

In the design review it is stated we need to have the same severity as CA2017 which is `warning`